cmd/inventory.go

```diff
@@ -53,6 +49,19 @@ func inventory(ctx context.Context) error {
 		return err
 	}
 
+	metricsPusher := metrics.NewPusher(logger, "inventory")
+	metricsPusher.AddCollector(collectors.NewGoCollector())
+	err = metricsPusher.Start()
+	if err != nil {
+		return err
+	}
+
+	otelCtx, otelShutdown := otelinit.InitOpenTelemetry(ctx, "fleet-scheduler")
+	defer otelShutdown(ctx)
+
+	otelCtxWithCancel, cancelFunc := context.WithCancel(otelCtx)
```

Comment:
Why do you need this second context? Isn't `otelCtx` a context with a cancelation function (specifically `otelShutdown`)?

Reply:
Maybe I dont understand enough about context, but I cant find anything within otelinit that cancels the ctx, only shuts down opentelemetry.

Reply:
That `otelShutdown` function is [bogus](https://github.com/equinix-labs/otel-init-go/blob/14a1d0e47a1b1228f2ca3572a7421bd94e9ff595/otelinit/public.go#L36).